File: CodeWars/.js/8kyu-Even-or-Odd.js
// Create a function that takes an integer as an argument and returns "Even" for even numbers or "Odd" for odd numbers.

const chai = require('chai');
const assert = chai.assert;

describe("Sample tests",() => {
  
  it("2 is even", () => {
    assert.strictEqual(even_or_odd(2), "Even");
  });
  it("7 is odd", () => {
    assert.strictEqual(even_or_odd(7), "Odd");
  });
  it("-42 is even", () => {
    assert.strictEqual(even_or_odd(-42), "Even");
  });
  it("-7 is odd", () => {
    assert.strictEqual(even_or_odd(-7), "Odd");
  });
  it("0 is even", () => {
    assert.strictEqual(even_or_odd(0), "Even");
  });
});


// my code

function even_or_odd(number) {
    if (number % 2 == 0){
      return('Even');
    }
    else{
      return('Odd');
    }
  }

File: CodeWars/.cs/8kyu-EvenOrOdd.cs
// Create a function that takes an integer as an argument and returns "Even" for even numbers or "Odd" for odd numbers.

using System;
using NUnit.Framework;

namespace Solution
{
  [TestFixture]
  public class SolutionTest
  {
    [Test]
    public void ExampleTest()
    {
      Assert.AreEqual("Even", SolutionClass.EvenOrOdd(2));
      Assert.AreEqual("Odd", SolutionClass.EvenOrOdd(1));
      Assert.AreEqual("Even", SolutionClass.EvenOrOdd(0));
      Assert.AreEqual("Odd", SolutionClass.EvenOrOdd(7));
      Assert.AreEqual("Odd", SolutionClass.EvenOrOdd(-1));
    }
  }
}

// my code

namespace Solution
{
  public class SolutionClass
  {
    public static string EvenOrOdd(int number) => (number % 2 == 0) ? "Even" : "Odd";
  }
}